pkg/apis/core/v1beta1/types_shoot.go

```diff
@@ -63,6 +63,9 @@ type ShootSpec struct {
 	// Addons contains information about enabled/disabled addons and their configuration.
 	// +optional
 	Addons *Addons `json:"addons,omitempty" protobuf:"bytes,1,opt,name=addons"`
+	// CloudProfile contains a reference to a CloudProfile or a NamespacedCloudProfile.
+	// +optional
+	CloudProfile *CloudProfileReference `json:"cloudProfile,omitempty" protobuf:"bytes,22,opt,name=cloudProfile"`
```

Comment:
Please move this down again. While I understand the motivation, the protobuf number ordering gets messed up this way.

Reply:
I have dropped the respective commit. Either dashboard needs to sort then or we wait until the Shoot api is promoted to stable v1.